inventory/upcloud.py

```diff
@@ -68,7 +68,11 @@
 import os
 import sys
 import argparse
-import ConfigParser
+try:
```

Comment:
This is needed to support both python3 and python2.7. In python3 configparser was moved to six.moves.